packages/orchestration/src/examples/shared/chain-hub-cf.js

```diff
@@ -0,0 +1,50 @@
+import { M } from '@endo/patterns';
+import { heapVowE } from '@agoric/vow/vat.js';
+import { CosmosChainInfoShape } from '../../typeGuards.js';
+
+/**
+ * @import {Zone} from '@agoric/zone';
+ * @import {CosmosChainInfo, IBCConnectionInfo} from '@agoric/orchestration';
+ * @import {ChainHub} from '../../exos/chain-hub.js';
+ */
+
+/**
+ * @param {Zone} zone
+ * @param {ChainHub} chainHub
+ */
+export const prepareChainHubCreatorFacet = (zone, chainHub) => {
```

Comment:
while this is useful for some contracts to return as a `creatorFacet`, that's not what this is.

please pick a name that closely describes what this Exo does. consider `ChainHubAdmin`, `ChainHubHelper` or `ChainAdder`.